src/clr-angular/data/datagrid/providers/selection.spec.ts

```diff
@@ -311,6 +313,20 @@ export default function(): void {
                     expect(selectionInstance.isSelected(items[5])).toBe(true);
                 });
 
+                it("trackBy function is not invoked when the " +
+                       "items are not populated by the selection is defined",
```

Comment:
This sentence makes no sense. 😮

Reply:
lol. it doesn't sorry. i always forget to check this before submitting a PR. i was having difficulties writing the test and changed this multiple times. updating now.

Reply:
how about now?